pages/mesosphere/dcos/1.14/gui/settings/index.md

```diff
@@ -4,14 +4,117 @@ navigationTitle:  Settings
 title: Settings
 menuWeight: 10
 excerpt: Using the Settings menu
+enterprise: true
 render: mustache
 model: /mesosphere/dcos/1.14/data.yml
 ---
 
-In the Settings menu you can manage package repositories, change the UI settings, add and manage LDAP directories, and add and manage identity providers.
+From the **Settings** tab, you can manage 
 
+- UI Settings
+- Package repositories
+- Secret Stores
+- LDAP directories
+- Identity providers
 
-
-![Package repositories](/mesosphere/dcos/1.14/img/GUI-Settings-EE-Package_Repositories-1_12.png)
+![Package repositories](/mesosphere/dcos/1.14/img/GUI-Settings-Package-Repositories.png)
 
 Figure 1 - **Settings > Package Repositories** tab
+
+# UI Settings
+
+The **UI Settings** tab allows you to manage your DC/OS UI version and the language in which the UI is presented.
+
+## DC/OS UI Details
+
+The **DC/OS UI Details** section shows you which version of the UI is installed; note that this is **not** the version of DC/OS which is currently installed. You have the option to use an earlier version of the UI. 
+
+Click on the **Rollback** button to load the earlier version of the UI.
```

Comment:
DC/OS comes with a pre-bundled version. As a uses updates the UI using the settings page we keep the pre-bundled version in case we need to roll back. So should a user roll back they will be getting the pre-bundled version that came with their dc/os installation.
It is indeed an `earlier` version. Just a bit more context.

Reply:
Thanks. I've added this to the doc.